packages/components/src/Dropdown/Dropdown.tsx

```diff
@@ -48,6 +55,23 @@ type DropdownProps = ListboxProps & {
    * Optional className for additional styling.
    */
   className?: string;
+  /**
+   * Render dropdown button that is only as wide as the content.
+   *
+   * When defining compact dropdown need to provide optionsWidth to
+   * define width of options menu, and optionally provide optionsAlign
+   * if desired to right align to dropdow button.
```

Comment:
Got a little typo in there.
```suggestion
   * if desired to right align to dropdown button.
```

Reply:
Thanks for catching. I am removing that particular documentation actually.